pygmt/src/meca.py

```diff
@@ -221,6 +221,33 @@ def meca(  # noqa: PLR0912, PLR0913, PLR0915
     r"""
     Plot focal mechanisms.
 
+    The following focal mechanism conventions are supported:
+
+    .. list-table:: Supported focal mechanism conventions
+       :widths: 15 15 70
+       :header-rows: 1
+
+       * - Convention
+         - Description
+         - Focal parameters
+       * - ``"aki"``
+         - Aki and Richard
+         - *strike*, *dip*, *rake*, *magnitude*
+       * - ``"gcmt"``
+         - global centroid moment tensor
+         - *strike1*, *dip1*, *rake1*, *strike2*, *dip2*, *rake2*,
+           *mantissa*, *exponent*
+       * - ``"mt"``
+         - seismic moment tensor
+         - *mrr*, *mtt*, *mff*, *mrt*, *mrf*, *mtf*, *exponent*
+       * - ``"partial"``
+         - partial focal mechanism
+         - *strike1*, *dip1*, *strike2*, *fault_type*, *magnitude*
+       * - ``"principal_axis"``
+         - principal axis
+         - *t_value*, *t_azimuth*, *t_plunge*, *n_value*, *n_azimuth*, *n_plunge*,
+           *p_value*, *p_azimuth*, *p_plunge*, *exponent*
```

Comment:
Maybe it's time to add more notes for the parameters (see https://docs.generic-mapping-tools.org/dev/supplements/seis/meca.html). For example,

- Angles are in degrees, not radians
- moment components are in dynes-cm
- fault_type means +1/-1 for normal/reverse fault
- ...